### DISCUSSION TITLE
Nothing happens after entering pin

### DISCUSSION THREAD

**Discussion by spidermike151** [iotaledger/firefly#3462] [Help & Support] [answered]
I've looked for answers, tried a few things but cannot figure this one out, upon install of newest firefly version I get to the enter pin screen, I enter my pin and hit enter and nothing happens, it wont take me to my wallet or do anything, I'm not the brightest with computer stuff so before you say uninstall, reinstall, and enter with seed phrase I've tried uninstalling but everytime I reinstall it just goes right back to the enter pin screen, any help would be mucho appreciado!

**Comment by riffensen** [comment]
I'm having a similar issue getting into firefly wallet , only I enter my Pin and it says Incorrect ( I am sure the Pin is correct)

**Reply by MarWar69** [reply]
I have a similar problem,
After entering my Pin code, the password / Page does not enter. setup is complete but no password is available  therefore can not send or receive money :(

**Comment by kweelulaay** [comment]
Passcode is correct but can't login to account in my PC how ?

**Comment by MiKeF05** [comment, marked as answer]
It seems that the PIN saved in the keychain of the OS gets lost. You can just create a new profile and use the option to recover your wallet

**Reply by spidermike151** [reply]
Thanks brother, and apologies to everyone else with similar issues, just as I posted this topic i found this suggestion elsewhere and it really is that simple of a fix, a tad annoying but not too hard at all.
